app/views/index/index.html.erb

```diff
@@ -1,3 +1,18 @@
+<% if @email %>
+<div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+  <div class="govuk-notification-banner__header">
+    <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+      Notice
```

Comment:
I can't check what this will look like easily as it's not on test atm (as far as I can see) - I'm not sure if this Notice title pops up or what the banner looks like and I'm assuming it will only show for logged in users because it is in an `@email` conditional.

Reply:
That's right. I have pushed some changes after reviewing with Patrick. Can I get a re-review please?